### PR TITLE
Rev dependencies to cats-effect 1.0, circe 0.10, cats.1.4, http4s 0.1…

### PR DESCRIPTION
**Pull request by sbuzzard**
…9.  Make TraceT stack safe (or at least stacksafe when the underlying effect is a Monad), added typeclass tests, added ability to use either monotonic or real time time in order to measure span duration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Quick links:
 - [Examples of use](#usage)
 - [Configuration](#config)
 - [How to get latest version](#getit)
-- API Docs [Core](https://oss.sonatype.org/service/local/repositories/releases/archive/com/ccadllc/cedi/dtrace-core_2.12/1.4.0/dtrace-core_2.12-1.4.0-javadoc.jar/!/com/ccadllc/cedi/dtrace/index.html) [Logging](https://oss.sonatype.org/service/local/repositories/releases/archive/com/ccadllc/cedi/dtrace-logging_2.12/1.4.0/dtrace-logging_2.12-1.4.0-javadoc.jar/!/com/ccadllc/cedi/dtrace/logging/index.html) [Logstash](https://oss.sonatype.org/service/local/repositories/releases/archive/com/ccadllc/cedi/dtrace-logstash_2.12/1.4.0/dtrace-logstash_2.12-1.4.0-javadoc.jar/!/com/ccadllc/cedi/dtrace/logstash/index.html)
```
Comment (mpilquist) [inline README.md:9]: Intentional?

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ import org.http4s._
 import org.http4s.dsl.io._
 
 import com.ccadllc.cedi.dtrace._
-import com.ccadllc.cedi.dtrace.interop.http4s._
+import com.ccadllc.cedi.dtrace.interop.htt4s._
```
Comment (mpilquist) [inline README.md]: Typo?

File: README.md [hunk 3]
```diff
@@ -186,47 +196,47 @@ Cedi Distributed Trace supports Scala 2.11 and 2.12. This distribution is publis
 This is the core functionality, recording trace and span information over effectful programs, passing these recorded events to registred emitters for disposition.
 
 ```scala
-libraryDependencies += "com.ccadllc.cedi" %% "dtrace-core" % "1.4.0"
+libraryDependencies += "com.ccadllc.cedi" %% "dtrace-core" % "1.3.0-SNAPSHOT"
```
Comment (mpilquist) [inline README.md]: Intentional?

File: build.sbt [hunk 4]
```diff
@@ -1,6 +1,10 @@
-lazy val circeVersion = "0.9.3"
+lazy val catsEffectVersion = "1.0.0"
 
-lazy val http4sVersion = "0.18.11"
+lazy val catsCoreVersion = "1.4.0"
+
+lazy val circeVersion = "0.10.0"
+
+lazy val http4sVersion = "0.19.0-M2"
```
Comment (mpilquist) [inline build.sbt]: M3 was released this morning according to Ross

File: README.md (hunk 1 above)
Comment (sbuzzard) [reply README.md:9]: no, updated in latest.

Comment (mpilquist) [reply README.md:9]: Looks good but missing the logstash one still

**Comment by mpilquist** [comment]
I released cedi-build 1.1.0 btw -- that's why Travis is failing

File: README.md (hunk 1 above)
Comment (sbuzzard) [reply README.md:9]: sigh.  yes.  And money.  and http4s.  and xb3.

**Pull request merged**